src/main/java/paymelah/logic/parser/ParserUtil.java

```diff
@@ -160,7 +166,32 @@ public static Money parseMoney(String money) throws ParseException {
     }
 
     /**
-     * Returns true if none of the prefixes contains empty {@code Optional} values in the given
+     * Parses {@code String s} into a {@code NameContainsKeywordsPredicate}.
+     */
+    public static NameContainsKeywordsPredicate parseNameContainsKeywordsPredicate(String s) throws ParseException {
```

Comment:
Should the method names be prepareXContainsKeywordsPredicate() instead? I feel like that reads better